### PR TITLE
ci: add separate job for smoketests

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
CI

**Did you add tests for your changes?**
No.

**If relevant, did you update the documentation?**
No.

**Summary**

Add different jobs for smoke tests.

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

**Comment by alexander-akait** [comment]
Let's run sometesks only on the minimum node.js version, no need to do a lot of tests for this stuff

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -111,6 +108,33 @@ jobs:
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v3
 
+  smoketests:
+    name: Smoketests - ${{ matrix.os }} - Node v${{ matrix.node-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node-version: [10.x, 12.x, 14.x, 16.x, 18.x]
```
Comment (snitin315) [inline .github/workflows/nodejs.yml:117]:
```suggestion
        node-version: [10.x]
```

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -111,6 +108,33 @@ jobs:
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v3
 
+  smoketests:
+    name: Smoketests - ${{ matrix.os }} - Node v${{ matrix.node-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
```
Comment (snitin315) [inline .github/workflows/nodejs.yml:116]:
```suggestion
        os: [ubuntu-latest]
```

File: .github/workflows/nodejs.yml [hunk 3]
```diff
@@ -111,6 +108,33 @@ jobs:
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v3
 
+  smoketests:
+    name: Smoketests - ${{ matrix.os }} - Node v${{ matrix.node-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [10.x]
```
Comment (snitin315) [inline .github/workflows/nodejs.yml:117]:
```suggestion
        node-version: [14.x]
```

**Review by alexander-akait — APPROVED** [review]

**Comment by alexander-akait** [comment]
Thank you

**Pull request merged**